app/test/unit/git/commit-test.ts

```diff
@@ -138,9 +141,10 @@ describe('git/commit', () => {
 
       expect(files.length).toEqual(1)
 
-      await createCommit(repo, 'renamed a file', [
+      const sha = await createCommit(repo, 'renamed a file', [
         files[0].withIncludeAll(true),
       ])
+      expect(sha).toHaveLength(7)
```

Comment:
Just a warning on this magic number `7`: in a repository where there are lots of commits, the shortened commit ID here may be than 7 characters. For example, on the current `HEAD` of `desktop/desktop` I get a shortened SHA of 8 characters:

```shellsession
$ git rev-parse HEAD
0ce1ba23eff6da91ceedd2e01069c188faf85995
$ git rev-parse --short HEAD
0ce1ba23e
```

For newer repositories it's still likely to be 7 characters, and it's fine for tests to check for this length, but if we go and take this assumption into the application code we need to be aware of this feature of Git.

Reply:
👍